build_dir/MyFunctionbf3066d1DepLayer/AWS_SAM_CLI_README

```diff
@@ -0,0 +1 @@
+This layer contains dependencies of function MyFunction and automatically added by AWS SAM CLI command 'sam sync'
```

Comment:
Is this file needed?

Reply:
It is not lol, just another remnant from running `./Make -Pr` that I forgot to remove